main.py

```diff
@@ -670,22 +590,62 @@ def sizechanged(e):
     manager_button.configure(width=int(app.winfo_width() / 12))
 
 
-# THEMED IMAGES
+def make_windows_blur(window):
+    from ctypes import windll
+
+    from BlurWindow.blurWindow import GlobalBlur
+
+    GlobalBlur(
+        windll.user32.GetParent(window.winfo_id()),

```

Comment:
`windll.user32.GetParent(window.winfo_id())` is reliable, because it will always give the id of our tkinter window, even if it happens not to be the 'ForegroundWindow'.